Assets/MRTK/SDK/Editor/Inspectors/UX/ObjectManipulator/ObjectManipulatorInspector.cs

```diff
@@ -188,6 +211,42 @@ public override void OnInspectorGUI()
                 EditorGUILayout.PropertyField(scaleLerpTime);
             }
 
+            EditorGUILayout.Space();
+            elasticsFoldout = EditorGUILayout.Foldout(elasticsFoldout, "Elastics", true);
+
+            if (elasticsFoldout)
+            {
+                // This two-way enum cast is required because EnumFlagsField does not play nicely with
+                // SerializedProperties and custom enum flags.
+                var newElasticTypesValue = EditorGUILayout.EnumFlagsField("Manipulation types using elastic feedback: ", (TransformFlags)elasticTypes.intValue);
+                elasticTypes.intValue = (int)(TransformFlags)newElasticTypesValue;
+
+                // If the particular elastic type is requested, we offer the user the ability
+                // to configure the elastic system.
+                TransformFlags currentFlags = (TransformFlags)elasticTypes.intValue;
+
+                translationElasticFoldout = DrawElasticConfiguration<ElasticConfiguration>("Translation Elastic",
```

Comment:
I was told otherwise in a prior PR by @wiwei (I think it was Will). The guidance was that the first line is ordinarily indented, with each argument beyond that vertically aligned with the first argument. I can clarify

Reply:
Ok, chatted with Will, deferring to this guidance

Reply:
Fixed